website/templates/get_oauth_email.mako

```diff
@@ -0,0 +1,24 @@
+<%inherit file="base.mako"/>
+<%def name="title()">OSF ORCID Login</%def>
```

Comment:
All instances of `ORCID` in this file should probably be replaced with something like `${oauth_provider}`, to generalize this template